dependent-service-downtime-and-ps-adaptor-behavior.md

```diff
@@ -0,0 +1,64 @@
+## Service Failures and Expected Behavior
+
+This guide outlines the expected behavior of the PS Adaptor when various dependent services are down. 
+The PS Adaptor works in conjunction with several services including the GP2GP Translator, MHS Adaptor, Facade, and others. 
```

Comment:
This is confusing, the Translator and Facade are parts of the PS Adaptor?! This sentence suggests that they are separate. 😕

Reply:
ok, I will correct it